packages/driver/src/dom/selection.ts

```diff
@@ -575,19 +575,25 @@ const _moveSelectionTo = function (toStart: boolean, el: HTMLElement, options =
         $elements.callNativeMethod(doc, 'execCommand', 'selectAll', false, null)
       }
     } else {
-      let range
-
-      // Sometimes, selection.rangeCount is 0 when there is no selection.
-      // In that case, it fails in Chrome.
-      // We're creating a new range and add it to the selection to avoid the case.
-      if (selection.rangeCount === 0) {
-        range = doc.createRange()
-        selection.addRange(range)
+      // WebKit must use selectAllChildren to ensure selections are made in all use cases.
+      // Some content, like HTML markup, cannot be selected using a range when inside a contenteditable field.
+      if (Cypress.isBrowser('webkit')) {
+        selection.selectAllChildren(el)
```

Comment:
Interesting, I wonder why we aren't just using `selectAllChildren` in all browsers, did you check if this works for Cr/FF?

Reply:
Good call, it does work for the current versions of Chrome/Firefox and Chromium 74, which is the earliest I can grab. Minimum supported Chrome is 64, so there's still a range of supported versions I can't validate.

Reply:
MDN says it's supported since uh.. Chrome... *squints* Chrome 1. https://developer.mozilla.org/en-US/docs/Web/API/Selection/selectAllChildren#browser_compatibility So it should be safe if the tests pass?

Reply:
Wow, straight from the primordial soup. Well we know it won't throw at least! I'll get that change made and verified in CI.